File: app/src/main/java/com/example/adam/myapplication/ui/doctor/doctor_tabs/DoctorPagerAdapter.java
package com.example.adam.myapplication.ui.doctor.doctor_tabs;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

import com.example.adam.myapplication.ui.doctor.create.DoctorTaskFragment;
import com.example.adam.myapplication.ui.doctor.doctors_list_tab.DoctorsFragment;

public class DoctorPagerAdapter extends FragmentPagerAdapter {

    private final String[] TAB_TITLES = {"TASK", "DOCTORS", "HISTORY"};

    DoctorPagerAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        switch (position) {
            case 0:
                return new DoctorTaskFragment();

            case 1:
                return new DoctorsFragment();

            case 2:
                return new DoctorTaskFragment();
        }

        return null;
    }

    @Override
    public CharSequence getPageTitle(int position) {
        return TAB_TITLES[position];
    }

    @Override
    public int getCount() {
        return 3;
    }
}

File: app/src/main/java/com/example/adam/myapplication/ui/doctor/tabs/DoctorPagerAdapter.kt
package com.example.adam.myapplication.ui.doctor.tabs

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter

import com.example.adam.myapplication.ui.doctor.create.DoctorTaskFragment
import com.example.adam.myapplication.ui.doctor.list.DoctorsFragment

class DoctorPagerAdapter internal constructor(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    companion object {
        private val TITLES = arrayOf("TASK", "DOCTORS", "HISTORY")
    }

    override fun getItem(position: Int): Fragment? {
        when (position) {
            0 -> return DoctorTaskFragment()

            1 -> return DoctorsFragment()

            2 -> return DoctorTaskFragment()
        }

        return null
    }

    override fun getPageTitle(position: Int): CharSequence? {
        return TITLES[position]
    }

    override fun getCount(): Int {
        return 3
    }
}
